packages/core/package.json

```diff
@@ -68,6 +68,7 @@
     "@rollup/plugin-replace": "^6.0.2",
     "@rollup/plugin-typescript": "^12.1.2",
     "@types/jest": "^26",
-    "rollup": "^4.24.0"
+    "rollup": "^4.24.0",
+    "tslib": "^2.8.1"
```

Comment:
apparent missing dependency for usage of spread. Not sure exactly why this has arisen now